src/main/java/org/dependencytrack/tasks/vulnerabilitypolicy/blobstorage/S3StorageHandler.java

```diff
@@ -0,0 +1,65 @@
+package org.dependencytrack.tasks.vulnerabilitypolicy.blobstorage;
+
+import alpine.common.logging.Logger;
+import io.minio.BucketExistsArgs;
+import io.minio.GetObjectArgs;
+import io.minio.GetObjectResponse;
+import io.minio.MinioClient;
+import io.minio.StatObjectArgs;
+import io.minio.StatObjectResponse;
+import io.minio.errors.ErrorResponseException;
+import io.minio.errors.InsufficientDataException;
+import io.minio.errors.InternalException;
+import io.minio.errors.InvalidResponseException;
+import io.minio.errors.ServerException;
+import io.minio.errors.XmlParserException;
+import org.dependencytrack.tasks.vulnerabilitypolicy.AbstractVulnerabilityPolicyFetcher;
+import org.dependencytrack.tasks.vulnerabilitypolicy.S3Client;
+import org.dependencytrack.util.VulnerabilityPolicyUtil;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.util.zip.ZipInputStream;
+
+public class S3StorageHandler extends AbstractVulnerabilityPolicyFetcher {
+    public static S3Client S3CLIENT = new S3Client();
```

Comment:
What's the reasoning behind making this `public static` instead of `private final` (and not `static`)? AFAICT this variable is exclusively used internally in this class.